servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/ConnectionAcceptorOffloadingTest.java

```diff
@@ -15,60 +15,187 @@
  */
 package io.servicetalk.http.netty;
 
-import io.servicetalk.concurrent.api.Single;
+import io.servicetalk.concurrent.api.Completable;
 import io.servicetalk.http.api.BlockingHttpClient;
-import io.servicetalk.http.api.HttpRequest;
 import io.servicetalk.http.api.HttpResponse;
-import io.servicetalk.http.api.HttpResponseFactory;
 import io.servicetalk.http.api.HttpResponseStatus;
-import io.servicetalk.http.api.HttpServiceContext;
+import io.servicetalk.http.api.HttpServerBuilder;
+import io.servicetalk.http.api.HttpService;
 import io.servicetalk.transport.api.ConnectExecutionStrategy;
 import io.servicetalk.transport.api.ConnectionAcceptorFactory;
+import io.servicetalk.transport.api.ConnectionInfo;
+import io.servicetalk.transport.api.EarlyConnectionAcceptor;
 import io.servicetalk.transport.api.IoThreadFactory;
+import io.servicetalk.transport.api.LateConnectionAcceptor;
 import io.servicetalk.transport.api.ServerContext;
 
+import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
 
 import java.net.SocketAddress;
+import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicReference;
 
 import static io.servicetalk.concurrent.api.Single.succeeded;
 import static io.servicetalk.http.api.HttpSerializers.textSerializerUtf8;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.notNullValue;
 import static org.hamcrest.core.Is.is;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 class ConnectionAcceptorOffloadingTest {
```

Comment:
Consider additional tests to make sure all acceptors work with HTTP/1.1 and HTTP/2 and with/without SslConfig. It will help to verify that `EarlyConnectionAcceptorHandler` works as expected.
If you'd like, we can aggregate all tests under `ConnectionAcceptorTest` class in http-netty instead of splitting into `ConnectionAcceptorOffloadingTest` and `ConnectionAcceptorRejectionTest`.